File: 3195-separate-black-and-white-balls/3195-separate-black-and-white-balls.cs
public class Solution {
    public long MinimumSteps(string s) {
        long res = 0;
        long swap = 0;
        foreach (var ch in s) {
            if (ch == '1')
                swap++;
            else
                res += swap;
        }
        return res;
    }
}

File: 3195-separate-black-and-white-balls/3195-separate-black-and-white-balls.py
class Solution(object):
    def minimumSteps(self, s):
        """
        :type s: str
        :rtype: int
        """
        res, swaps = 0, 0
        for ch in s:
            if ch == '1':
                swaps += 1
            else:
                res += swaps
        return res 